plugins/commands/ps/command.rb

```diff
@@ -0,0 +1,116 @@
+require "optparse"
+
+require_relative "../../communicators/winrm/helper"
+
+module VagrantPlugins
+  module CommandPS
+    class Command < Vagrant.plugin("2", :command)
+      def self.synopsis
+        "connects to machine via powershell remoting"
+      end
+
+      def execute
+        options = {}
+
+        opts = OptionParser.new do |o|
+          o.banner = "Usage: vagrant ps [-- extra ps args]"
```

Comment:
I'm trying to use the same convention as `vagrant ssh` which uses this same OptionParser banner language. This basically does the same thing: the extra ps args are simply passed through to the powershell.exe command.